src/browser/service/routing.rs

```diff
@@ -87,7 +87,13 @@ pub fn setup_link_listener(notify: impl Fn(Notification) + 'static) {
         event.target()
             .and_then(|et| et.dyn_into::<web_sys::Element>().ok())
             .and_then(|el| el.closest("a[href]").ok().flatten())
-            .and_then(|href_el| href_el.get_attribute("href"))
+            .and_then(|href_el| {
+                if href_el.get_attribute("download").is_none () {
```

Comment:
Please change the the line:
```suggestion
                if href_el.has_attribute("download") {
```
There are some potential [problems](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#non-existing_attributes) with `get_attribute` and `has_attribute` better describes our intentions and there is a chance it's slightly faster.